benchmarks/dynamo/runner.py

```diff
@@ -360,8 +365,9 @@ def generate_commands(args, dtypes, suites, devices, compilers, output_dir):
         lines.append("#!/bin/bash")
         lines.append("set -x")
         lines.append("# Setup the output directory")
-        lines.append(f"rm -rf {output_dir}")
-        lines.append(f"mkdir {output_dir}")
+        if not args.keep_output_dir:
```

Comment:
You would just need to put "rm" command in the if statement? Assume you would still need "mkdir {output_dir}" in both cases? 

I'm also thinking would  mkdir {output_dir}_{dtypes} just work.

Reply:
i.e. just change {output_dir} to {output_dir}_{dytpes} and there would probably be no need for keep_output_dir.

Reply:
Sure, I can keep `mkdir {output_dir}`.  It works both ways because the directory has already been created earlier https://github.com/pytorch/pytorch/blob/master/.ci/pytorch/test.sh#L289

On the other hand, `{output_dir}` would need to be set to `WORKSPACE/test/test-reports` so that it can be picked up by the CI in which the convention is to upload that directory containing the test reports.  I could also use `{dtypes}` subdirectory, but then I think I should keep the behavior of the script the same, just in case.  The script runs the same way when the new flag is not set.